### PR TITLE
Unstable bcdbtests

### PR DESCRIPTION
**Pull request by despiegk**

### CODE REVIEW

File: JumpscaleCore/data/bcdb/tests/11_redis_index_lost.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def test_redis_index_lost():
 
     """
 
-    bcdb = j.data.bcdb.get("test", reset=True)
+    bcdb = j.data.bcdb.get("test_redis_index", reset=True)
```
Comment (despiegk) [inline JumpscaleCore/data/bcdb/tests/11_redis_index_lost.py]: please don't make distinction, its not needed
bcdb needs to be cleaned using the storage layer underneath like it was

File: JumpscaleCore/data/bcdb/tests/11_redis_index_lost.py [hunk 2]
```diff
@@ -68,3 +68,10 @@ def test_redis_index_lost():
 
     j.data.bcdb._log_info("TEST DONE")
     return "OK"
+
+
+# Teardown
+def after():
```
Comment (despiegk) [inline JumpscaleCore/data/bcdb/tests/11_redis_index_lost.py]: no tear down, remove everywhere

**Review by despiegk — COMMENTED** [review]
please make all changes, let kristof review

**Comment by xmonader** [comment]
needs to be readdressed later.

**Pull request closed without merging**